packages/@adobe/spectrum-css-temp/components/tabs/index.css

```diff
@@ -169,6 +167,7 @@ governing permissions and limitations under the License.
     block-size: var(--spectrum-tabs-rule-height);
 
     inset-block-end: calc(var(--spectrum-tabs-rule-height) * -1);
+    transition: transform var(--spectrum-tabs-selection-indicator-animation-duration) ease-in-out, width var(--spectrum-tabs-selection-indicator-animation-duration) ease-in-out;
```

Comment:
still a bit iffy (the line will still show some weird transition behavior w/ regards to sizing), can be solved by increasing the animation duration (300 ms works pretty nice). Loath to change it since it has a var tied to it though